app/src/main/java/cm/aptoide/pt/view/DeepLinkManager.java

```diff
@@ -44,10 +49,15 @@
 import rx.Completable;
 import rx.Observable;
 import rx.android.schedulers.AndroidSchedulers;
+import rx.subjects.PublishSubject;
+import rx.subscriptions.CompositeSubscription;
+
+import static cm.aptoide.pt.DeepLinkIntentReceiver.FROM_SHORTCUT;
 
 public class DeepLinkManager {
 
   public static final String DEEPLINK_KEY = "Deeplink";
+  private static final String APPS_SHORTCUTS = "App_Shortcut";
```

Comment:
Shouldn't the string name be APP_SHORTCUT?

Reply:
Fixed